src/components/Header/Header.css

```
@@ -1,41 +1,42 @@
-.logo {
+.header {
   position: absolute;
-  height: 25%;
-  width: 25%;
-  right: 8%;
-  top: 1%;
-  transform: rotate(22deg);
+  display: flex;
+  justify-content: space-around;
+  height: 150px;
+  width: 100%;
+  padding-top: 10px;
+  margin-bottom: 5vh;
+}
+
+.title-logo-link {
+  width: 50%;
```

```suggestion
  width: 16%;
```
Updates the link width to fit to the size of the logo img